themes/beaver/layouts/page/free-consultation.html

```diff
@@ -88,7 +88,7 @@ <h1 class="fl-heading">
                                                       <div class="pp-icon-wrapper animated none">
                                                          <div class="pp-infolist-icon">
                                                             <div class="pp-infolist-icon-inner">
-                                                               <img decoding="async" src="{{ absURL "wp-content/uploads/2022/10/tick-circle-white.svg" }}" alt="" role="presentation" />

```

Comment:
+1

Reply:
I don't see this code in this branch

Reply:
Added finger printing. For it needed to move icon to other folder (

Later, imo, better make bullet list css to use svg, instead of img tag

Reply:
reverted to not mix this with extract image resizing